apps/site/lib/site/realtime_schedule.ex

```diff
@@ -90,6 +98,21 @@ defmodule Site.RealtimeSchedule do
     |> List.flatten()
   end
 
+  defp get_alert_counts(route_with_patterns, now, alerts_fn) do
+    route_with_patterns
+    |> Enum.map(fn {_stop_id, route, _patterns} ->
+      {route.id,
+       Task.async(fn -> get_high_priority_alert_count_for_route(route.id, now, alerts_fn) end)}
+    end)
+    |> Enum.into(%{}, fn {route_id, task} -> {route_id, Task.await(task)} end)
```

Comment:
We've ignored this pronto error in the past week -- I looked at it briefly and I think the efficiency issue doesn't apply to elixir 1.8, so this is yet another good reason to ditch pronto for an update to date credo. 

I _think_ updated credo checks the elixir version for this warning.